src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -65,4 +70,13 @@ private function getExpressionLanguage()
 
         return $this->expressionLanguage;
     }
+
+    private function getPropertyAccessor()
+    {
+        if (!class_exists('Symfony\Component\PropertyAccess\PropertyAccess')) {
+            throw new RuntimeException('Unable to use expressions with data path as the Symfony PropertyAccess component is not installed.');
```

Comment:
I wonder if we should not throw this exception in the constraint to allow to detect this mistake earlier.

Reply:
Of course we could throw it there. But than we need to check there if `dataPath` is set because only then we check if the PropertyAccess is avaibale

Reply:
I think that would be fine.